src/Android/Android.csproj

```diff
@@ -168,7 +163,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\App\App.csproj">
-      <Project>{9F1742A7-7D03-4BB3-8FCD-41BC3002B00A}</Project>
+      <Project>{EE44C6A1-2A85-45FE-8D9B-BF1D5F88809C}</Project>
```

Comment:
I am confused about why this changed. Isn't this the project's ID?

Reply:
VSmac did that when it restructured the other parts of this file after the move to Forms 4.5.  Some searching shows it's apparently just VSmac being VSmac - I'll test it with the original value...

Reply:
Didn't seem to phase anything, reverted.